plugins/Installation/FormDatabaseSetup.php

```diff
@@ -302,7 +303,7 @@ private function dropExtraTables($db)
 }
 
 /**
- * Filename check for prefix/DB name
+ * Filename check for prefix
  *
  */
 class FormDatabaseSetup_Rule_checkValidFilename extends HTML_QuickForm2_Rule
```

Comment:
maybe table prefix also can't contain `+`, if so:  should we also validate the table prefix with the `DbHelper::isValidDbname`?